### PR TITLE
refactor: Account setting pages for all roles

### LINKED ISSUE #580: Account setting pages for all roles

### PR DESCRIPTION
**Pull request by whDongRui**
Resolve #580

### CODE REVIEW

File: src/components/Layout/index.jsx [hunk 1]
```diff
@@ -96,25 +91,32 @@ class Layout extends Component {
...
     const { isNormal, isDev, isAdmin } = this.props.user;
     const hasMenu = (isDev || isAdmin) && !isHome;
-    const paths = ['/dashboard', '/profile', '/ssh_keys'];
-    const hasSubNav = hasMenu && !noSubMenu && !hasBack && !paths.includes(match.path);
+    const hasSubNav = hasMenu && !noSubMenu && !hasBack;
+
+    if (isNormal) {
+      return (
+        <div className={classnames(styles.layout, className)}>
+          {noNotification ? null : <Notification />}
```
Comment (iwisunny) [inline src/components/Layout/index.jsx]: `!noNotification && <Notification />`

File: src/pages/Account/SSHKeys/columns.js [hunk 2]
```diff
@@ -12,7 +12,9 @@ export default (clusters, t) => [
     title: t('Name'),
     key: 'name',
     width: '130px',
-    render: item => <TdName name={item.name} description={item.node_id} noIcon />
```
Comment (iwisunny) [inline src/pages/Account/SSHKeys/columns.js]: No need to change.

File: src/pages/Account/index.jsx [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+            name="email"
+            value={userDetail.email}
+            disabled
+            readOnly
+          />
+        </div>
+        <div>
+          <label className={styles.name}>{t('Language setting')}</label>
+          <Select onChange={changeLanguage} value={userStore.language}>
+            <Select.Option value="zh">简体中文</Select.Option>
```
Comment (iwisunny) [inline src/pages/Account/index.jsx]: Add translation

File: src/components/Layout/SideNav/navMap.js [hunk 4]
```diff
@@ -163,13 +163,13 @@ export const getDevSubNavs = appId => [
...
-    name: 'Account info',
-    link: '/profile',
+    name: 'Account Info',
+    link: '/dashboard/account',
     iconName: 'folder'
   },
   {
     name: 'Change Password',
-    link: '/profile',
+    link: '/dashboard/account?type=Change Password',
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/components/Layout/SideNav/navMap.js]: ok, the accuout url will change '/dashboard/account/:type'

File: src/components/Layout/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/components/Layout/index.jsx]: OK, this previous code.

**Pull request merged**